src/guides/v2.3/reference/mftf/action-groups.md

```diff
@@ -0,0 +1,27 @@
+---
+group: mftf
+---
+
+# MFTF action group reference
+
+Action groups are important building blocks for quickly creating tests for the Magento Functional Testing Framework.
+This page lists all current action groups so developers can see what is available to them.
+
+## Action group list
+
+{% assign actiongroups = site.data.codebase.mftf.action-groups | group_by: "module" | sort: "name"  %}
+{% for item in actiongroups %}
+
+### {{ item.name }}
```

Comment:
This makes the list to be a list of files, not the list of action groups.
Consider making this an action group name. The file name with a link to the source code would go then to metadata.
Add sorting for the action groups.

Consider the same for functional tests.